lib/administrate/constants.rb

```diff
@@ -0,0 +1,3 @@
+DEFAULT_INDENT = 2
+ONE = 1
+TWO = 2
```

Comment:
Yeah I wasn't sure how to handle this, listing the constants at the top of the `routes generator` file seemed odd. But yeah having them top level object is bad - something I will try to avoid, 

Also not sure if literal `1` and `2` would be ok...

Reply:
Just to clarify: the constants should go inside the module, not just at the top of the file. Otherwise they'll also be assigned to `Object`.

As for the literals... yes, they would be ok :-) (But if you can find a good name for the `TWO` one, that'd be good too).